### ISSUE TITLE
Support for frame-ancestors

### ISSUE THREAD

**Issue by mattdell** [jagaapple/next-secure-headers#45]
# 🌱 Feature Request
## Is your feature request related to a problem? Please describe.
A clear and concise description of what you want and what your use case is.

I'm looking to set the CSP frame-ancestors header: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors

as such: 

```
Content-Security-Policy: frame-ancestors 'self' https://www.example.org;
```

## Describe the solution you'd like

I would like to be able to set in the configuration a key value such as...

```
frameAncestors: "'self' https://www.example.org"
```

## Describe alternatives you've considered
I'm currently doing this...

```
    headers: async () => [
      {
        source: '/(.*)',
        headers: [
          ...createSecureHeaders(),
          {
            key: 'Content-Security-Policy',
            value: "frame-ancestors 'self' *.example.com;",
          },
        ]
      },
    ],
```

## Documentation, Adoption, Migration Strategy
If you can, explain how users will be able to use this and how it might be documented. Maybe a mock-up?

## Additional context

---

- [ ] I've tried to find similar issues and pull requests
- [x] I would like to work on this feature 💪🏻

**Comment by mattdell** [comment]
Sorry if it's premature but I've raised a PR as #46

**Comment by jagaapple** [comment]
This will be released v2.2.0.
#41 #46
